paseos/actors/actor_builder.py

```diff
@@ -305,6 +306,39 @@ def set_position(actor: BaseActor, position: list):
         actor._position = position
         logger.debug(f"Setting position {position} on actor {actor}")
 
+    @staticmethod
+    def set_geometric_model(
+            actor: SpacecraftActor,
+            mass: float,
+            vertices=None,
+            faces=None,
+            scale: float=1
+
+    ):
+        """Define geometry of the spacecraft actor.
+
+        Args:
+            actor (SpacecraftActor): Actor to update.
+            mass (float): Mass of the spacecraft in kg
+            vertices (list): List of all vertices of the mesh
+            faces (list): List of the indexes of the vertices of a face
```

Comment:
It is not very clear how faces are linked to vertices without accessing Trimesh documentation. Please improve comment.